### PR TITLE
save pdfs to temp folder

### PR DESCRIPTION
**Pull request by mdayican**
### Change description ###

save pdfs to temp folder

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipFileProcessor.java [hunk 1]
```diff
@@ -35,11 +59,14 @@ public ZipFileProcessingResult process(
...
+                    log.info(
+                        "File: {}, Meta data size: {}",
+                        zipFileName,
+                        FileUtils.byteCountToDisplaySize(metadata.length)
+                    );
                     break;
                 case "pdf":
-                    pdfs.add(new Pdf(zipEntry.getName(), toByteArray(extractedZis)));
-
+                    pdfs.add(new Pdf(zipEntry.getName(), null));
```
Comment (apereverzin) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipFileProcessor.java:69]: Names and sizes of pdfs should also be logged - here or after the while loop.

Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipFileProcessor.java:69]: we do not have the size here, I will refactor this processor soon.

Comment (apereverzin) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipFileProcessor.java:69]: at least the names should be logged.

**Review by apereverzin — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipFileProcessor.java (hunk 1 above)
Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipFileProcessor.java:70]: name and size added to related method.

**Comment by mdayican** [comment]
stuck

**Pull request closed without merging**